app/schemas.py

```
@@ -460,6 +479,39 @@ class NotifierBase(BaseModel):
     )
     smtp_use_tls: Optional[bool] = Field(True, description="Use TLS encryption (default: True)")
 
+    @validator("address")
+    @classmethod
+    def validate_address(cls, v, values):
```

Ruff (0.14.13)

Missing return type annotation for classmethod `validate_address`

(ANN206)